sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -294,3 +292,22 @@ export interface Assessment {
   totalProvincialContribution: number;
   otherAllowableCost: number;
 }
+
+/**
+ * Interface for PartTime assessment payload.
+ */
+export interface PartTimeAssessment extends BaseAssessment {
+  schoolFees: number;
+  miscellaneousCost: number;
+  totalFamilyIncome: number;
+  totalAssessmentNeed: number;
+}
+/**
+ * This is a type which provides the contract for FullTime and PartTime assessment payload
+ * which is stored to database by workflow.
+ * It is possible that more properties can be added to the assessment payload
+ * without updating this interface and displayed in NOA form.
```

Comment:
is it `without updating this interface`? or `with`?

Reply:
Without @ann-aot.
While in runtime, the values will flow from Camunda->DB->API->NOA without need for a API code.
It does not mean that a developer should not add new properties here, it just means that the Camunda can generate new properties to be saved on the jsonB and these new properties will reach the form.io definition.

Reply:
As the column in jsonb, we can add additional columns and display.

Reply:
mm..okay